proposals/0275-allow-more-characters-like-whitespaces-and-punctuations-for-escaped-identifiers.md

```diff
@@ -52,7 +83,7 @@ identifier → ` identifier-head identifier-characters opt `
 with:
 ```
 identifier → ` escaped-identifier `
-escaped-identifier -> Any Unicode scalar value except U+000A or U+000D or U+0060
+escaped-identifier -> Any Unicode scalar value except U+000A, U+000B, U+000C, U+000D, U+0085, U+2028, U+2029 or U+0060 (back-tick)
```

Comment:
Could you please add a small description next to the rest of the Unicode characters. For example: `U+0085 (nel)` or `U+0085 (next line)`, just like you did with U+0060.